### ISSUE TITLE
WordPress Health Check recommends imagick

### ISSUE THREAD

**Issue by lwj5** [bitnami/bitnami-docker-wordpress#257]
**Description**
WordPress Site Health Status recommends imagick.

```
One or more recommended modules are missing
Performance
PHP modules perform most of the tasks on the server that make your site run. Any changes to these must be made by your server administrator.

The WordPress Hosting Team maintains a list of those modules, both recommended and required, in the team handbook (opens in a new tab).

Warning The optional module, imagick, is not installed, or has been disabled.
```

**Steps to reproduce the issue:**

1. Run
2. Admin > Tools > Health Check
3. Above appears recommended improvements

**Describe the results you received:**
A recommendation

**Describe the results you expected:**
No recommendation. Imagick is installed in the container.

**Version**

- Output of `docker version`:

```
-
```

- Output of `docker info`:

```
-
```

- Output of `docker-compose version` (if applicable):

```
-
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
AWS

**Comment by miguelaeh** [comment]
Hi @lwj5 ,
The `imagick` module is included in the image but not enabled by default.  To enable it you can use the following command:

```
sed -i 's/;extension = imagick/extension = imagick/' /opt/bitnami/php/etc/php.ini
```

Then to check the results:

```
$ php -i | grep magick
imagick
imagick module => enabled
imagick module version => 3.4.4
imagick classes => Imagick, ImagickDraw, ImagickPixel, ImagickPixelIterator, ImagickKernel
```

**Comment by lwj5** [comment]
@miguelaeh I see. So to put it in CMD, the command is as such. Is this right?

`/bin/bash -c '. /opt/bitnami/base/functions && . /opt/bitnami/base/helpers && . /apache-init.sh && . /wordpress-init.sh && nami_initialize apache mysql-client wordpress && . /post-init.sh && sed -i "s/;extension = imagick/extension = imagick/" /opt/bitnami/php/etc/php.ini && httpd -f /opt/bitnami/apache/conf/httpd.conf -DFOREGROUND'`

**Comment by marcosbc** [comment]
Hi @lwj5, you could fork the container and edit the app-entrypoint to perform those changes. However, currently there isn't any other way (init scripts mounted to /docker-entrypoint-init.d) would not be valid since they only execute once).

However, it looks like we should be bundling/enabling imagick with the Bitnami WordPress container. I've created an internal task so we work on it, but I cannot give you an ETA.

In the meantime, we assume this is already working for you as per your previous comments, let us know if it is not the case.

**Comment by lwj5** [comment]
Yes, it's working with the command above. Thanks for the help guys.

**Comment by marcosbc** [comment]
Hi @lwj5, we just released `bitnami/wordpress:5.7.1-debian-10-r22` with the fix, amongst [many other improvements](https://github.com/bitnami/bitnami-docker-wordpress#571-debian-10-r21) (released in the previous revision, `-r21`).
